akka-cluster-sharding/build.sbt

```diff
@@ -0,0 +1,18 @@
+import akka.{ AkkaBuild, Dependencies, Formatting, MultiNode, Unidoc, OSGi }
+import com.typesafe.tools.mima.plugin.MimaKeys
+
+AkkaBuild.defaultSettings
+
+Formatting.formatSettings
+
+Unidoc.scaladocSettingsNoVerificationOfDiagrams
+
+Unidoc.javadocSettings
+
+OSGi.clusterSharding
```

Comment:
as a general remark: should we not inline these settings here instead of keeping them centralized?